modules/Bio/EnsEMBL/Analysis/Hive/Config/EnsemblAnnoHelixer_conf.pm

```diff
@@ -507,8 +524,8 @@ sub pipeline_analyses {
           cmd => 'MAIN_LINES=$(wc -l < "#rnaseq_summary_file#"); GENUS_LINES=$(wc -l < "#rnaseq_summary_file_genus#"); ' .
                 'if [ "#override_evidence_threshold#" -eq 1 ]; then ' .
                 '  if [ $MAIN_LINES -ge 1 ]; then exit 1; else exit 2; fi; ' .
-                'elif [ $MAIN_LINES -ge ' . $self->o('rnaseq_main_file_min_lines') . ' ]; then exit 1; ' .
-                'elif [ $MAIN_LINES -lt ' . $self->o('rnaseq_main_file_min_lines') . ' ] && [ $GENUS_LINES -ge ' . $self->o('rnaseq_genus_file_min_lines') . ' ]; then exit 2; ' .
+                'elif [ $MAIN_LINES -ge #rnaseq_main_file_min_lines# ]; then exit 1; ' .
```

Comment:
Just to note on these changes they are for a fix covered in #521

the code in experimental/gbiab apparently never worked